3rdParty/fuerte/include/fuerte/message.h

```diff
@@ -193,8 +194,13 @@ class Request final : public Message {
   /// @brief request header
   RequestHeader header;
 
+
   MessageType type() const override { return MessageType::Request; }
   MessageHeader const& messageHeader() const override { return header; }
+  void setFuzzReqHeader(std::string& fuzzHeader) {_fuzzReqHeader = fuzzHeader;}
```

Comment:
The string is passed by (mutable) reference, and then it is copied. That looks strange. Is that intentional?

Reply:
switched the argument to const&